### PR TITLE
Micah jank - Feature Update #6 - BirthdayBot Scheduler

### PR DESCRIPTION
**Pull request by MicahJank**
Filled out and created the scheduling command for the birthdaybot via Feature Request #6

**Issues**:
1. Discord allows unverified users to join a server. I don't think they are assigned a discord ID so if an unverified user has a birthday the bot will tag them as undefined.

### CODE REVIEW

File: src/commands/birthday.js [hunk 1]
```diff
@@ -27,14 +27,16 @@ module.exports = {
         console.log(member)
         if(member){
             let memberIndex = dbServer.members.findIndex(m => m.user === message.author.tag)
-            dbServer.members[memberIndex].birthday = date       
+            dbServer.members[memberIndex].birthday = date
+            dbServer.members[memberIndex].discord_id = message.author.id
```
Comment (MicahJank) [inline src/commands/birthday.js:31]: discord_id is neccessary for the bot to properly tag the user when wishing them a happy birthday so I updated the birthday.js file to add the users discord id when they create their birthday

**Review by MicahJank — COMMENTED** [review]
added some clarifying comments

**Comment by MicahJank** [comment]
Fixed conflicts with base branch

**Comment by tomassirio** [comment]
Amazing job @MicahJank. Join us in the [discord channel ](discord.gg/h67z6e)so you are up to date with the issues of the bots if you want to

**Comment by MicahJank** [comment]
> Amazing job @MicahJank. Join us in the [discord channel ](discord.gg/h67z6e)so you are up to date with the issues of the bots if you want to

@tomassirio That discord link doesnt seem to be working for me?

**Comment by tomassirio** [comment]
Ups! try this [link](https://discord.gg/wtrfB7)

**Pull request merged**